.github/scripts/github_utils.py

```diff
@@ -128,7 +128,7 @@ def gh_fetch_json_dict(
 
 def gh_graphql(query: str, **kwargs: Any) -> dict[str, Any]:
     rc = gh_fetch_url(
-        "https://api.github.com/graphql",
+        "https://api.github.com/graphql",  # @lint-ignore
```

Comment:
Which lint?

Reply:
There's another PR https://github.com/pytorch/pytorch/pull/151844 introducing the linter, and this one fixes everything broken preliminary